config/routes.rb

```diff
@@ -1,3 +1,13 @@
 Rails.application.routes.draw do
   root to: "pages#index"
+  
+  mount ActionCable.server => '/cable'
+
+  unless Rails.env.production?
+    namespace :test do
+      namespace :components do
+        get 'iterations_summary_table/:id', to: 'iterations_summary_table#index', as: 'iterations_summary_table'
+      end
+    end
+  end
```

Comment:
These will be routes that are used by the tests. The idea is that each view will just render the relevant component and then the relevant test can test interactions with that test. I've also opened them to dev so that you can see and play with the components in dev mode, without having to find them in context.

This should all encourage good decoupling of components from each other and other view-level dependencies.